Automattic/wp-calypso
# Previews: Open external sites in the same window

Trying to prevent many opened tabs at once.

To test:

- My Sites
- Click "View Site"
- Click the external icon in top right corner
- It should now open in the **same tab**
